### PR TITLE
releases: add 4.0.0-rc release notes

### PR DESCRIPTION
**Pull request by toutdesuite**
### What is changed, added or deleted? (Required)

add 4.0.0-rc release notes

### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

**If you select two or more versions from above**, to trigger the bot to cherry-pick this PR to your desired release version branch(es), you **must** add corresponding labels such as **needs-cherry-pick-4.0**, **needs-cherry-pick-3.1**, **needs-cherry-pick-3.0**, and **needs-cherry-pick-2.1**.

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### CODE REVIEW

**Comment by scsldb** [comment]
LGTM

File: releases/4.0.0-rc.md [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+
+# TiDB 4.0 RC Release Notes
+
+发版日期：2020 年 04 月 08 日
+
+TiDB 版本：4.0.0-rc
+
+TiDB Ansible 版本：4.0.0-rc
+
+## TiDB 4.0.0-rc 兼容性变化
```
Comment (lilin90) [inline releases/4.0.0-rc.md:14]: Sounds redundant.
```suggestion
## 兼容性变化
```

File: releases/4.0.0-rc.md [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+category: Releases
+---
+
+# TiDB 4.0 RC Release Notes
+
+发版日期：2020 年 04 月 08 日
+
+TiDB 版本：4.0.0-rc
+
+TiDB Ansible 版本：4.0.0-rc
```
Comment (lilin90) [inline releases/4.0.0-rc.md:12]: Remove?

File: releases/4.0.0-rc.md [hunk 3]
```diff
@@ -0,0 +1,106 @@
...
+    - 悲观事务支持 pipelined 功能，TPC-C 性能提升 20%， 风险：pipelined 功能可能会在执行阶段加锁不成功导致事务提交失败 [#6984](https://github.com/tikv/tikv/pull/6984)
+    - 调整 unify-read-pool 配置项的方式，仅在新部署的集群时默认启用，旧集群保持原来的方式 [#7059](https://github.com/tikv/tikv/pull/7059)
+
++ Tools
+
+    - TiDB Binlog
+
+        * 新增验证 Common Name 配置项目的功能 [#934](https://github.com/pingcap/tidb-binlog/pull/934)
+
+## TiDB 4.0.0-rc 重点修复的 Bug
```
Comment (lilin90) [inline releases/4.0.0-rc.md:31]:
```suggestion
## 重点修复的 Bug
```

File: releases/4.0.0-rc.md [hunk 4]
```diff
@@ -0,0 +1,106 @@
...
+        * 修复 MySQL sink 执行 DDL 时，若下游没有 test 库系统报错的问题 [#353](https://github.com/pingcap/ticdc/pull/353)
+        * CDC cli 新增实时交互模式功能 [#351](https://github.com/pingcap/ticdc/pull/351)
+        * 同步数据时增加对上游表是否可同步的检查 [#368](https://github.com/pingcap/ticdc/pull/368)
+        * 新增异步写入 Kafka 的功能 [#344](https://github.com/pingcap/ticdc/pull/344)
+
++ TiDB Ansible
+
+    - 优化 CPU epollexclusive 检查失败时提示信息，包括：通过升级内核版本解决，且提示支持的最小内核版本 [#1230](https://github.com/pingcap/tidb-ansible/pull/1230)
+    - 优化 Dashboard 初始化的方式，修复升级到 4.0 版本失败的问题 [#1237](https://github.com/pingcap/tidb-ansible/pull/1237)
+    - 优化 TiFlash 目录创建的方式 [#1239](https://github.com/pingcap/tidb-ansible/pull/1239)
```
Comment (lilin90) [inline releases/4.0.0-rc.md:106]: Remove?

**Comment by TomShawn** [comment]
@scsldb @lilin90 Comments addressed, PTAL again, thanks!

**Pull request merged**